doc/examples/segmentation/plot_watershed.py

```diff
@@ -54,7 +54,7 @@
 
 ax0.imshow(image, cmap=plt.cm.gray, interpolation='nearest')
 ax0.set_title('Overlapping objects')
-ax1.imshow(-distance, cmap=plt.cm.jet, interpolation='nearest')
+ax1.imshow(-distance, cmap=plt.cm.gray, interpolation='nearest')
```

Comment:
What do you mean?

![figure_1](https://cloud.githubusercontent.com/assets/58611/12710396/554dabb4-c884-11e5-958c-ecdaefe69551.png)